### DISCUSSION TITLE
/learner/data/v1/system/settings/get/sunbird - 404 error

### DISCUSSION THREAD

**Discussion by akulavasu** [Sunbird-Ed/Community#681] [Sunbird ED configuration discussion] [answered]
Hello @gandham-santhosh @santhosh-tg @AmiableAnil,

Upon loading the homepage, the request to https://sunbird.ex001.in/learner/data/v1/system/settings/get/sunbird gives status code 404

![image](https://github.com/Sunbird-Ed/Community/assets/34706681/56ecac12-aebd-4bba-b8c3-6e30889a4e41)

Request to https://sunbird.ex001.in/learner/data/v1/system/settings/get/custodianOrgId works as expected.

![image](https://github.com/Sunbird-Ed/Community/assets/34706681/cea45c60-037e-48e0-819c-c714ae8b06d7)

Please let us know if any configuration needs to be changed for custodian org and channel.

**Comment by gandham-santhosh** [comment]
@akulavasu was setSystemSettings API ran as part of functional configuration successful?

**Reply by akulavasu** [reply]
@gandham-santhosh below APIs were successful. Are they the ones for system settings

![image](https://github.com/Sunbird-Ed/Community/assets/34706681/fddc8f64-4fc5-465c-84cc-7b3abf67c49a)

Values provided are,
custodianOrgid - 0139826110739742726
custodianRootOrgid - 0139826110739742726
custodianOrgChannel - sunbird

**Reply by AmiableAnil** [reply, marked as answer]
@akulavasu Please find the sample response below. You need to set for your setup. If it's not set or for any system setting, you are trying to read but it's not set, it will give 404 error.
`"result": {
        "response": {
            "id": "sunbird",
            "field": "sunbird",
            "value": "{\\\"latestVersion\\\":\\\"v1\\\",\\\"v1\\\":{\\\"url\\\":\\\"https:\/\/dev-sunbird-temp.azureedge.net\/portal\/terms-and-conditions-v1.html\\\"}}"
        }
    }`
